memory/cloudsql/chat_message_history.go

```diff
@@ -225,3 +225,28 @@ func (c *ChatMessageHistory) Messages(ctx context.Context) ([]llms.ChatMessage,
 
 	return messages, nil
 }
+
+// SetMessages clears the current messages from the ChatMessageHistory for a
+// given session and then adds new messages to it.
+func (c *ChatMessageHistory) SetMessages(ctx context.Context, messages []llms.ChatMessage) error {
+	if !c.overwrite {
+		return nil
+	}
```

Comment:
not sure I follow this behavior -- if overwrite is not true we don't SetMessages?

I suspect this isn't the behavior we want, but if it is we should probably return an error and add a comment here.

Reply:
I removed the overwrite checks